modules/apps/commerce/commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/portlet/action/EditCommerceShipmentItemMVCActionCommand.java

```diff
@@ -228,9 +228,10 @@ private CommerceShipmentItem _updateCommerceShipmentItem(
 		CommerceShipmentItem commerceShipmentItem = null;
 
 		List<CommerceInventoryWarehouse> commerceInventoryWarehouses =
-			_commerceInventoryWarehouseService.getCommerceInventoryWarehouses(
-				commerceOrderItem.getCompanyId(),
-				commerceOrderItem.getGroupId(), true);
+			_commerceInventoryWarehouseLocalService.
+				getCommerceInventoryWarehouses(
```

Comment:
it shouldn't be need to use local service after https://github.com/liferay-commerce/liferay-portal/pull/3933

Reply:
https://github.com/liferay-commerce/liferay-portal/pull/3912/commits/ce2a08bef97a89e8e169889092fa8495692cf14f